### PR TITLE
Add navbar back button and implement on required views

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js [hunk 1]
```diff
@@ -4,20 +4,31 @@
...
+
             $scope.$on('nih.notifications.hide', hideAlert);
             $scope.$on('nih.notifications.show', showAlert);
+            $scope.$on('$stateChangeSuccess', function (event, toState) {
+                history.push(toState);
+            });
+        }
+
+        function back() {
+            var stateName = history.length > 1 ? history.splice(-2)[0].name : 'profiles';
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js]: A note for what the indexing is about here might be helpful.

File: src/nih_wayfinding/app/index.html [hunk 2]
```diff
@@ -28,7 +28,10 @@
         </div>
 
         <div class="navbar-header pull-right">
-          <div class="btn-group" dropdown>
+          <button ng-if="navbar.config.back" type="button" class="btn btn-primary" ng-click="navbar.back()">
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/index.html:31]: Is this what we have in the wireframes?  Seems like back buttons usually go on the left, so putting on the right might be confusing.

**Comment by flibbertigibbet** [comment]
+1

File: src/nih_wayfinding/app/index.html (hunk 2 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/index.html:31]: Yup, this matches the wireframes. I expect Jeff will be moving things around.

File: src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js]: Yeah, this code is kinda lame. I'll tweak it a bit.

**Pull request closed without merging**